Update Export LS-DYNA keyword file (#800)
ansys/pyprimemesh: doc/source/user_guide/fileio.rst

   method allows you to export IGA LS-DYNA keyword ``(*.k)`` files and set parameters for exporting boundary-fitted
   splines using the :class:`ExportBoundaryFittedSplineParams <ansys.meshing.prime.ExportBoundaryFittedSplineParams>` class.

 - The :func:`FileIO.export_lsdyna_keyword_file() <ansys.meshing.prime.FileIO.export_lsdyna_keyword_file>`
   method allows you to write out LS-DYNA Keyword ``(*.k)`` file that contains mesh definition and other 
   necessary information to carry out the analysis run by LS-DYNA solver. 
   :class:`ExportLSDynaKeywordFileParams <ansys.meshing.prime.ExportLSDynaKeywordFileParams>` class allows 
   you to specify the application type (SEATBELT, DOORSLAM), compute the spot weld thickness or not, 
   append the material cards in the k file, provide the database cards to append in the k file,
   and allows you to specify the LS-DYNA data field format. You should specify the material 

>    properties card and the database keywords card in LS-DYNA format.

Reviewer: @srpyprime we should clearly mark this as beta.

   method allows you to export IGA LS-DYNA keyword ``(*.k)`` files and set parameters for exporting boundary-fitted
   splines using the :class:`ExportBoundaryFittedSplineParams <ansys.meshing.prime.ExportBoundaryFittedSplineParams>` class.

 - The :func:`FileIO.export_lsdyna_keyword_file() <ansys.meshing.prime.FileIO.export_lsdyna_keyword_file>`

>    method allows you to write out LS-DYNA Keyword ``(*.k)`` file that contains mesh definition and other 

Reviewer 2: ```suggestion
   function allows you to write out an LS-DYNA Keyword ``(*.k)`` file that contains the mesh definition and other 
```
Given that the ``:func`` role is used, I changed the word from method to function.

>    necessary information to carry out the analysis run by LS-DYNA solver. 
>    :class:`ExportLSDynaKeywordFileParams <ansys.meshing.prime.ExportLSDynaKeywordFileParams>` class allows 

Reviewer 2: ```suggestion
   necessary information to carry out the analysis run by the LS-DYNA solver. 
   The :class:`ExportLSDynaKeywordFileParams <ansys.meshing.prime.ExportLSDynaKeywordFileParams>` class allows 
```

   you to specify the application type (SEATBELT, DOORSLAM), compute the spot weld thickness or not, 
   append the material cards in the k file, provide the database cards to append in the k file,
   and allows you to specify the LS-DYNA data field format. You should specify the material 
   properties card and the database keywords card in LS-DYNA format.

 .. note::

>     :func:`FileIO.export_lsdyna_keyword_file() <ansys.meshing.prime.FileIO.export_lsdyna_keyword_file>` is a beta API. The behavior and implementation might change in the future.

Reviewer 2: ```suggestion
    The :func:`FileIO.export_lsdyna_keyword_file() <ansys.meshing.prime.FileIO.export_lsdyna_keyword_file>` function
    is a beta API. The behavior and implementation might change in the future.
```

   method allows you to export IGA LS-DYNA keyword ``(*.k)`` files and set parameters for exporting boundary-fitted
   splines using the :class:`ExportBoundaryFittedSplineParams <ansys.meshing.prime.ExportBoundaryFittedSplineParams>` class.

 - The :func:`FileIO.export_lsdyna_keyword_file() <ansys.meshing.prime.FileIO.export_lsdyna_keyword_file>`
   method allows you to write out LS-DYNA Keyword ``(*.k)`` file that contains mesh definition and other 
   necessary information to carry out the analysis run by LS-DYNA solver. 
   :class:`ExportLSDynaKeywordFileParams <ansys.meshing.prime.ExportLSDynaKeywordFileParams>` class allows 
   you to specify the application type (SEATBELT, DOORSLAM), compute the spot weld thickness or not, 
   append the material cards in the k file, provide the database cards to append in the k file,
   and allows you to specify the LS-DYNA data field format. You should specify the material 
   properties card and the database keywords card in LS-DYNA format.

>  .. note::
>     :func:`FileIO.export_lsdyna_keyword_file() <ansys.meshing.prime.FileIO.export_lsdyna_keyword_file>`
>    is a beta API. The behavior and name may change in the future.

Reviewer: ```suggestion
 .. note::
    :func:`FileIO.export_lsdyna_keyword_file() <ansys.meshing.prime.FileIO.export_lsdyna_keyword_file>` is a beta API. The behavior and implementation might change in the future.
```
